src/state/get-drag-impact/in-home-list.js

```diff
@@ -28,7 +28,9 @@ export default ({
   insideHome,
 }: Args): DragImpact => {
   const axis: Axis = home.axis;
-  const homeScrollDiff: Position = subtract(home.scroll.current, home.scroll.initial);
+  const homeScrollDiff: Position = subtract(
```

Comment:
Was the test case for this file updated?

Reply:
Nothing changed here functionally, I just had to update it because the shape of `DroppableDimension` changed.

Reply:
:thumbsup: